$OpenBSD: patch-mk_build_mk,v 1.3 2003/08/12 13:25:18 avsm Exp $
SplitObjs are unnecessary (it splits object files up), and seems
to use enormous amounts of memory, eventually stomping on my
stack/data/file size ulimits when set, and dumping core. Solution
: turn it off

Help ghc find libgmp.a

--- /dev/null	Tue Aug  5 10:47:30 2003
+++ mk/build.mk	Tue Aug  5 10:51:30 2003
@@ -0,0 +1,2 @@
+SplitObjs=NO
+SRC_HC_OPTS+=-L$(LOCALBASE)/lib
